docs/source/conf.py

```diff
@@ -110,8 +100,7 @@ def get_authors_from_citation_file(file_name):
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
-# source_suffix = ['.rst', '.md']
-source_suffix = ".rst"
+source_suffix = ['.rst']
```

Comment:
I recall the discussions regarding using markdown but I dont recall the conclusion of the discussions. With this change here, I assume we now decide to keep only the rst files, right?

Reply:
This is the "preparation" for it. I will add '.md' as soon as we actually add a markdown file.